src/scribe.js

```diff
@@ -71,8 +72,10 @@ define([
     var TransactionManager = buildTransactionManager(this);
     this.transactionManager = new TransactionManager();
 
-    var UndoManager = buildUndoManager(this);
-    this.undoManager = new UndoManager();
+    if (this.options.shouldUndo) {
+      var UndoManager = buildUndoManager(this);
+      this.undoManager = new UndoManager();
```

Comment:
In the cases where `scribe.undoManager` is called directly this makes sense, but thats not the only pieces of functionality that are changed. The assignment of event listeners are determined by this option and it makes no practical sense to check against the undoManager in this case. In order to be consistent therefore, we should always check against the option.